src/main/target/NAZE-rev6/AFROMINI.mk

```diff
@@ -0,0 +1 @@
+/home/christian/Development/betaflight/src/main/target/NAZE/AFROMINI.mk
```

Comment:
Sorry, my file explorer created absolute symbolic links instead of relative links.